parser.cpp

```diff
@@ -1185,9 +1194,9 @@ namespace Sass {
     { return new (ctx.mem) Null(pstate); }
 
     if (lex< identifier >()) {
-      String_Constant* str = new (ctx.mem) String_Constant(pstate, lexed);
+      String_Constant* str = new (ctx.mem) String_Quoted(pstate, lexed);
```

Comment:
Are you replacing all `String_Constant` with `String_Quoted`?

The semantics here are confusing since map keys don't have to be quoted.